src/Base.sol

```diff
@@ -3,6 +3,13 @@ pragma solidity >=0.8.0;
 
 library Base {
 
+    /******************************************************************************************************************/
+    /*** Governance Relay Addresses                                                                                 ***/
+    /******************************************************************************************************************/
+
+    address internal constant SPARK_EXECUTOR = 0xF93B7122450A50AF3e5A76E1d546e95Ac1d0F579;
```

Comment:
You need to revoke your deployer key as admin because the deploy script was incorrect.

Reply:
Revoked here https://basescan.org/tx/0xae30dfaaf43186f465e3c2b8d11a2d41fed798a7dce1437335d66a48f70bedf4